app/javascript/shared/helpers/AudioNotificationHelper.js

```diff
@@ -47,6 +47,10 @@ export const shouldPlayAudio = (
   userId,
   isDocHidden
 ) => {
+  if (!isDocHidden && message.message_type === MESSAGE_TYPE.INCOMING) {
```

Comment:
Use camelCase for message type

Reply:
Fixes by https://github.com/chatwoot/chatwoot/pull/4303/commits/99edefbd37bc0c1f0accfd32c96f261307922183